packages/pwa-kit-react-sdk/bin/pwa-build.js

```diff
@@ -53,4 +53,6 @@ sh.exec(`npm install --prefix build --no-save ${deps.join(' ')}`)
 
 sh.rm(`build/package.json`)
 
+sh.cp('package.json', 'build/')
```

Comment:
These two lines look a bit funny one after the other!! 😂

Reply:
Haha whoops.. fixed that.